core-services/secret-mirroring/_mapping.yaml

```diff
@@ -21,8 +21,20 @@ secrets:
     namespace: cincinnati-ci
     name: cincinnati-credentials
   to:
-    namespace: ci
+    namespace: test-credentials
```

Comment:
@stevekuznetsov can you confirm that this won't break existing jobs?

Reply:
It won't break it until you rotate the secret - the previous one will continue to exist. If that's OK, `/hold cancel` and we can merge.

/hold

Reply:
> It won't break it until you rotate the secret - the previous one will continue to exist. If that's OK, `/hold cancel` and we can merge.

That's ok, thanks!

/hold cancel